chats/apps/api/v1/tests/test_constraints.py

```diff
@@ -0,0 +1,85 @@
+from django.db import IntegrityError
+from rest_framework.test import APITestCase
+
+from chats.apps.projects.models import ProjectPermission
+from chats.apps.queues.models import Queue, QueueAuthorization
+from chats.apps.rooms.models import Room
+from chats.apps.sectors.models import Sector, SectorAuthorization, SectorTag
+
+
+class ConstraintTests(APITestCase):
+    fixtures = ['chats/fixtures/fixture_sector.json']
+
+    def setUp(self):
+       self.project_permission = ProjectPermission.objects.get(uuid="e416fd45-2896-43a5-bd7a-5067f03c77fa")
+       self.queue = Queue.objects.get(uuid="f2519480-7e58-4fc4-9894-9ab1769e29cf")
+       self.queue_auth = QueueAuthorization.objects.get(uuid="3717f056-7ea5-4d38-80f5-ba907132807c")
+       self.room = Room.objects.get(uuid="090da6d1-959e-4dea-994a-41bf0d38ba26")
+       self.sector = Sector.objects.get(uuid="21aecf8c-0c73-4059-ba82-4343e0cc627c")
+       self.sector_auth = SectorAuthorization.objects.get(uuid="e87a90ed-f217-4655-9116-5c0b51203851")
+       self.sector_tag = SectorTag.objects.get(uuid="62d9e7c4-4f2d-40fc-acf7-9549bface0fb")
+
+    def test_unique_user_permission_constraint(self):
+        with self.assertRaises(IntegrityError) as context:
+            project_permission = ProjectPermission.objects.create(
```

Comment:
the variable is not being used, in these cases _ is used in the variable.

in this discussion, although from another language they approach the subject well: https://stackoverflow.com/questions/70261169/why-we-are-using-prefix-in-some-cases-to-define-instance-variables-in -ruby